cli/azd/cmd/env.go

```diff
@@ -54,7 +55,7 @@ You can find all environment configurations under the *.azure\<environment-name>
 
 func envSetCmd(rootOptions *commands.GlobalCommandOptions) *cobra.Command {
 	actionFn := func(ctx context.Context, _ *cobra.Command, args []string, azdCtx *environment.AzdContext) error {
-		askOne := makeAskOne(rootOptions.NoPrompt)
+		console := input.NewAskerConsole(!rootOptions.NoPrompt)
```

Comment:
I suspect the `Console` interface we've introduced here is for mocking. We don't actually expect multiple implementations of `Console` out in production code.

I wonder if we should just rename `NewAskerConsole` as `NewConsole` for improved readability.

Reply:
Correct - the interface is to help aid in mocking/testing.  I will rename creation method to `NewConsole`.